cet_website_sale/views/website_sale_templates.xml

```diff
@@ -5,8 +5,9 @@
 
   <!-- Taken from oca/e-commerce/website_sale_tax_toggle and adapted for cet_website_sale -->
   <!-- Start of addition -->
+
   <template id="tax_toggle_template">
-    <t t-set="taxed" t-value="request.session.get('tax_toggle_taxed', request.env.user.with_context(skip_tax_toggle_check=True).has_group('account.group_show_line_subtotals_tax_included'))"/>
+    <t t-set="taxed" t-value="request.session.get('tax_toggle_taxed', request.env.user.with_context(skip_tax_toggle_check=True).has_group('sale.group_show_price_total'))"/>
```

Comment:
@vvrossem the `account.group_show_line_subtotals_tax_included` does not exist in v11 so I fixed to `sale.group_show_price_total`. It does not really solve the problem of the prices not being computed with the parameter "taxed" on first page load but at least, it's consistent :-)